manticore/platforms/evm.py

```diff
@@ -1153,6 +1166,7 @@ def _store(self, address, value):
         #CHECK VALUE IS A 256 BIT INT OR BITVEC
         self._allocate(address)
         self.memory.write(address, [value])
+        self._publish('did_evm_write_memory', address, value)
```

Comment:
this is not consistent with `_published_events` above?

Reply:
Yes  'evm_write_memory' is declared. It does not complain about that